Terminal.Gui/Drawing/Aligner.cs

```diff
@@ -152,7 +152,7 @@ public int ContainerSize
 
         for (var i = 0; i < sizes.Length; i++)
         {
-            CheckSizeCannotBeNegative (i, sizes);
+            CheckSizeCannotBeNegative (i, in sizes);
```

Comment:
Strike that `IEnumerable<T>` comment, since the original callee had `IReadOnlyList<T>`, which is probably better - if readonly semantics are mandatory.